.github/workflows/hotfix.yml

```diff
@@ -0,0 +1,22 @@
+name: Hotfix Production
+run-name: Triggered by ${{ github.actor }}
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Image tag to deploy. Defaults to the last commit SHA in the branch."
```

Comment:
👍  So this can roll back to a SHA older than the tip of main, e.g. if a deployment goes wrong and we want to just quickly undo it? Looks good.